### ISSUE TITLE
Additional installation steps and bug for CORD-19

### ISSUE THREAD

**Issue by DavidRivasPhD** [neuml/paperetl#6]
Hi David,
In addition to following your paperetl installation instructions I had to take these steps to get rid of the following error and warning:

![Screenshot from 2020-09-06 16-55-15](https://user-images.githubusercontent.com/61651965/92935823-8d8c8880-f406-11ea-9c8a-f223e20a8766.png)
```
~$ python3
>>> import nltk
>>> nltk.download(‘punkt’)
>>> exit()        
```                            
this created a directory **~/nltk_data/tokenizers/punkt**
    and fixed the above error

Also, the UserWarning below was eliminated as follows:
```
$ pip3 uninstall scikit-learn==0.23.2
$ pip3 install scikit-learn==0.23.1
```
![Screenshot from 2020-09-09 10-36-26](https://user-images.githubusercontent.com/61651965/92937149-35568600-f408-11ea-961e-ca3c7659bfaa.png)

Then, unfortunately after a full run the resulting articles.sqlite database came out with the Study Design fields, and Tags and Labels being **NULL** (see screenshots below).

![Screenshot from 2020-09-11 07-27-52](https://user-images.githubusercontent.com/61651965/92937426-8d8d8800-f408-11ea-8bae-2472b28794bb.png)

![Screenshot from 2020-09-11 07-29-24](https://user-images.githubusercontent.com/61651965/92937458-967e5980-f408-11ea-8ff3-bb3bedd68f11.png)

Any ideas on how solve this NULL issue would be appreciated

**Comment by davidmezzetti** [comment]
Hi David,

Unfortunately, NLTK has different models that are installed with different installations. Thank you for sharing that you needed to install an additional model.

Regarding study design, not all documents will have a detected study design (will be 0). Only certain documents will be tagged as COVID-19.

Please take a look at: https://www.kaggle.com/davidmezzetti/cord-19-etl and https://www.kaggle.com/davidmezzetti/cord-19-analysis-with-sentence-embeddings

These notebooks on Kaggle have all the background on how this process works.

**Comment by DavidRivasPhD** [comment]
Thank you David. After examining the results more carefully, unlike the above screenshots, 
I found plenty on non-NULL values (screenshots below); so the output is correct and there is no such a bug.  
Yes, I have read your writings on GitHub, Kaggle, and tds as well as Ankur Mohan’s blogs about your work. Your models are very nice, I have learned a lot from them.

![Screenshot from 2020-09-11 14-21-52](https://user-images.githubusercontent.com/61651965/92973082-9c902c80-f440-11ea-9ef1-82c95b948961.png)
![Screenshot from 2020-09-11 14-38-44](https://user-images.githubusercontent.com/61651965/92973094-a6b22b00-f440-11ea-8714-8736432a35e9.png)
![Screenshot from 2020-09-11 14-47-02](https://user-images.githubusercontent.com/61651965/92973097-ab76df00-f440-11ea-9c61-9ade6440d60c.png)
